lib/src/mt_tap.c

```diff
@@ -500,7 +500,8 @@ static int tap_bkg_thread(void* arg) {
     for (i = 0; i < num_ports; i++) {
       if (rx > 0 && pkts_rx[0]) {
         cni->tap_rx_cnt[i] += 1;
-        rte_eth_tx_burst(mt_port_id(impl, i), cni->tap_tx_q_id[i], pkts_rx, 1);
+        rte_eth_tx_burst(mt_port_id(impl, i), mt_dev_tx_queue_id(cni->tap_tx_q[i]),
```

Comment:
Use mt_dev_tx_burst instead. We may add some protect/statistics inthe mt_dev_tx_burst later.